docs/how-to-guides/auto-update-flatcar.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Auto-updating Flatcar Container Linux
+weight: 10
+---
+
+## Introduction
+
+Lokomotive supports Flatcar Container Linux only. Flatcar Linux Update Operator (FLUO) can keep the operating system updated every time there is a new Flatcar release. FLUO is a supported component of the Lokomotive's component stack. This document will show how to install it and disable its auto-update feature for specific nodes.
```

Comment:
Maybe we can link to FLUO repository here?

Also, I'd add line breaks.

Reply:
Also, "Lokomotive's component stack" sounds like a new term to me. Maybe we can just say "FLUO is one of the supported Lokomotive components"?

Reply:
`This document` -> `This guide`.